File: backend/src/main/java/com/backend/ord/domain/dto/BankDTO.java
package com.backend.ord.domain.dto;

import com.backend.ord.domain.dto.abstracts.DTOBase;
import lombok.*;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class BankDTO extends DTOBase {
    private String name;
    private String description;

    private UserDTO user;
    private BankGroupDTO group;
}

File: backend/src/main/kotlin/com/backend/ord/domain/dto/BankDTO.kt
package com.backend.ord.domain.dto

import java.time.Instant
import java.util.*

data class BankDTO(
    val id: UUID = UUID.randomUUID(),

    var name: String,
    var description: String,

    val user: UserDTO,
    var bankGroup: BankGroupDTO? = null,

    val createdAt: Instant = Instant.now(),
    var updatedAt: Instant = Instant.now(),
)
